src/globus_sdk/__init__.py

```diff
@@ -133,6 +133,11 @@ def _force_eager_imports() -> None:
         "TransferClient",
         "TransferData",
     },
+    "scopes": {
+        "Scope",
+        "ScopeParseError",
+        "ScopeCycleError",
+    },
```

Comment:
Should we be adding these as top-level exports? Or should we keep it at `globus_sdk.scopes.Scope`, as with `MutableScope`?

On the one hand, being exported at the top level is "simpler" -- on the other hand, it's already a very crowded namespace.